modules/heat_transfer/test/tests/physics/test_fe.i

```diff
@@ -0,0 +1,50 @@
+[Mesh]
+  [cmg]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = 10
+    dy = 10
+  []
+[]
+
+[Physics]
+  [HeatConductionFE]
+    [h1]
```

Comment:
Do we need this `[h1]` level?

In Griffin, we have a syntax like
```
[TransportSystems]
  # equation type, bc, source, etc.
  [sn]
    block = 0
    scheme = DFEM-SN
    # angular quadrature, FE family, order, etc.
  []
  # multiple sub-blocks are allowed with proper block restriction
[]
```
`[TransportSystems]` is very like `[HeatConduction]` but we use a meta action to add individual actions based on the `scheme` parameter.

Reply:
you do if you want to have multiple instances and want to vary those
```
  # equation type, bc, source, etc.
```
parameters

the nice thing is that you do have a 'discretization' block effectively. 

I think we could discuss whether it s better to have the discretization that way. it s a lot of nesting

```
[Physics]
  [Transport]
     [region_1_problem]
        block = 0
         # equation type, bc, source, etc.
         [FV]
            scheme = DFEM-SN
            # angular quadrature, FE family, order, etc.
          []
      []
    []
  []
  # multiple sub-blocks are allowed with proper block restriction
[]

Reply:
`equation type, bc, source, etc` are defining the problem and shared by all subdomains. Each subdomain has a particular discretization scheme. This is the reason why we designed `[TransportSystems]` syntax that way. The thermal conduction, we could potentially have CG+DG multi-scheme. We may have only part of the domain have heat conduction.

Reply:
I think it is fine to merge the equation or physics description with discretization as you did. We just do not need this extra input level `[h1]` here I think. It will be a little difficult to do multi-scheme but that may not be important.

Reply:
> radiation type (neutron/photon/etc), equation type (eigenvalue/transient), bc, source, etc are defining the problem and shared by all subdomains.

that s not the case for other Physics we have. 

> . The thermal conduction, we could potentially have CG+DG multi-scheme

we can already do that with the current syntax. 

We need the nested block for block restriction